tampermonkey-headers.js

```diff
@@ -1,10 +1,10 @@
 // ==UserScript==
 // @name        FUT Enhancer
 // @version     VERSION
-// @description Enhances the FIFA Ultimate Team 20 Web app. Includes Futbin integration and other useful tools
+// @description Enhances the FIFA Ultimate Team 21 Web app. Includes Futbin integration and other useful tools
 // @license     MIT
 // @author      Tim Klingeleers
-// @match       https://www.easports.com/fifa/ultimate-team/web-app/*
+// @match       https://www.ea.com/fifa/ultimate-team/web-app/*
```

Comment:
Needs another match on `https://www.ea.com/*/fifa/ultimate-team/web-app/*` to make sure localised versions are matched as well.

Reply:
ok, i'll add it